src/Tests/Creation/When_creating_forwarding_subscription.cs

```diff
@@ -57,7 +58,7 @@ public void TopicCleanUp()
             const string subscriptionName = "endpoint0";
             //make sure there is no leftover from previous test
             var namespaceManager = new NamespaceManagerAdapter(NamespaceManager.CreateFromConnectionString(AzureServiceBusConnectionString.Value));
-            await namespaceManager.DeleteSubscriptionAsync(topicPath, subscriptionName);
+            await namespaceManager.DeleteSubscription(new SubscriptionDescription(topicPath, subscriptionName));
```

Comment:
Why is the delete subscription contained in this PR? I suppose those changes are unrelated to this PR?